### DISCUSSION TITLE
Does scientific-python/cookie pass its own sp-repo-review tests?

### DISCUSSION THREAD

**Discussion by FrankHubeny** [scientific-python/cookie#472] [General]
While trying to make sure my own repo passes the review, I ran into problems getting **MY102: MyPy show_error_codes deprecated** to pass.  I am not using "show_error_codes".  It is the only failure that I get.  I don't know how to resolve it.

When I tried running the review against scientific-python/cookie, I see that MY102 also fails there.  Is there something wrong with the test?

Here is the url I used: https://learn.scientific-python.org/development/guides/repo-review/?repo=scientific-python%2Fcookie&branch=main

**Comment by henryiii** [comment]
This is an issue with the WASM version and I've never been able to trace down why it's happening. If you run it offline, in pre-commit, etc. it's fine. See https://github.com/scientific-python/cookie/issues/386.

**Reply by henryiii** [reply]
Ideas welcome! I think it's always the mypy checks.
